manifests/d/Decred/Decrediton/1.7.7/Decred.Decrediton.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Decred.Decrediton
+PackageVersion: 1.7.7
+PackageLocale: en-US
+Publisher: Decred

```

```suggestion
Publisher: Decred Team
```
Must match Apps&Features / Control Panel

---

See above.